### PR TITLE
cilium-cli: Define CLI_MAIN_DIR Make variable

### PR DESCRIPTION
**Pull request by michi-covalent**
Define CLI_MAIN_DIR Make variable to make it easier to override.

### CODE REVIEW

File: cilium-cli/Makefile [hunk 1]
```diff
@@ -10,6 +10,7 @@ BINDIR ?= /usr/local/bin
 CLI_VERSION=$(shell git describe --tags --always)
 THIS_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 CILIUM_VERSION=$(shell cat $(THIS_DIR)/../stable.txt)
+CLI_MAIN_DIR:=./cmd/cilium
```
Comment (aanm) [inline cilium-cli/Makefile]: shouldn't this be `CLI_MAIN_DIR ?= ./cmd/cilium` if we plan to override it while having a default?

**Review by aanm — CHANGES_REQUESTED** [review]

**Comment by michi-covalent** [comment]
@aanm ping! 🙏

**Review by aanm — APPROVED** [review]

**Comment by aanm** [comment]
@kkourt test

**Review by kkourt — APPROVED** [review]

**Review by asauber — APPROVED** [review]

**Pull request merged**